sql/install.sql

```diff
@@ -4,4 +4,31 @@ CREATE TABLE users (
   email TEXT,
   password VARCHAR(32),
   PRIMARY KEY (user_id)
-)
\ No newline at end of file
+);
+CREATE TABLE vocabulary (
+  vid INT NOT NULL AUTO_INCREMENT ,
+  vocabulary VARCHAR(32) NOT NULL ,
+  PRIMARY KEY (vid)
+ );
+INSERT INTO vocabulary (vid, vocabulary) VALUES (NULL, 'authors'), (NULL, 'categories');
+CREATE TABLE terms (
+  id INT NOT NULL AUTO_INCREMENT ,
+  vid INT,  name VARCHAR(64) NOT NULL UNIQUE ,
+  FOREIGN KEY (vid) REFERENCES vocabulary(vid),
+  PRIMARY KEY  (id)
+ );
+CREATE TABLE field_authors (
+  entity_id INT ,entity_type VARCHAR(32),
+  term_id INT  ,
+  FOREIGN KEY (entity_id) REFERENCES books(id),
```

Comment:
please remove foreign keys. Even if they make sense to be added, they will complicate things for this internship.